hazelcast/src/main/java/com/hazelcast/config/BaseMetricsConfig.java

```diff
@@ -53,8 +53,12 @@ protected BaseMetricsConfig(BaseMetricsConfig metricsConfig) {
      * enabled, Hazelcast Management Center will be able to connect to this
      * member. It's enabled by default.
      * <p>
-     * May be overridden by {@link ClusterProperty#METRICS_ENABLED}
-     * system property.
+     * May be overridden by {@link ClusterProperty#METRICS_ENABLED} system property for the member
+     * May be overridden by following system properties for the client.
+     * {@link com.hazelcast.client.properties.ClientProperty#METRICS_ENABLED}
+     * {@link com.hazelcast.client.properties.ClientProperty#STATISTICS_ENABLED}
+     * When both of them configured for the client STATISTICS_ENABLED is ignored.
+     *
      */
```

Comment:
```suggestion
     * May be overridden by {@link ClusterProperty#METRICS_ENABLED} system
     * property for the member.
     * <p>
     * May be overridden by {@link ClientProperty#METRICS_ENABLED} and 
     * {@link ClientProperty#STATISTICS_ENABLED} system properties for the 
     * client.
     * <p>
     * When both of them are configured for the client, 
     * {@link ClientProperty#STATISTICS_ENABLED} is ignored.
     */
```
Added some line breaks, used `ClientProperty` directly to have a shorter text (please add import at the top), removed the unnecessary empty line at the end.